dao/pom.xml

```diff
@@ -29,5 +33,39 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-maven-plugin</artifactId>
+            <version>${liquibase.version}</version>
```

Comment:
Please check if it is possible to add `<scope>test</scope>` as at dao level liquibase is used only to update the test database.